databases/prepare_clean_psql_wf11.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ ! "$WILDFLY_HOME" ]; then
+    printf "Error: Please configure the installation path for WildFly\n"
+    printf "   >>> Ex: WILDFLY_HOME=/path/to/your/wildfly11-installation\n"
+    exit 1
+else
+    printf "😂 Awesome, WildFly 11 is configured!\n"
+    printf "   >>> Preparing the PostgreSQL DB module\n"
+    cp -r ./src/main/resources/modules/org $WILDFLY_HOME/modules
```

Comment:
If this script is executed from another folder, passing the whole script path, this won't work.
I would suggest changing this to:
```bash
SCRIPTDIR=`dirname $0`
cp -r $SCRIPTDIR/src/main/resources/modules/org $WILDFLY_HOME/modules
```